### ISSUE TITLE
Base color not included

### ISSUE THREAD

**Issue by KevinBatdorf** [SimeonGriggs/tailwind-css-palette-generator#8]
This is the second tool I checked with a similar issue today so maybe there was a recent breaking change in a library somewhere. 

When I enter `#285e97` into the tool, that color is changed. Is this happening just for me?

<img width="1017" alt="Screen Shot 2020-12-12 at 6 47 58 PM" src="https://user-images.githubusercontent.com/1478421/101983032-98c3ae80-3caa-11eb-92f2-5e467b8221d8.png">

**Comment by SimeonGriggs** [comment]
Yeah there's something about the ways the colours are calculated that the input colour can come back with a different value.

...can spot the difference between #295f99 and #285e97? :D
 
<img width="1016" alt="Screen Shot 2020-12-12 at 4 45 54 pm" src="https://user-images.githubusercontent.com/9684022/101989635-a3297c80-3c99-11eb-93e5-84410a5b5f8d.png">

**Comment by KevinBatdorf** [comment]
> can spot the difference between #295f99 and #285e97? :D

I actually only noticed a few weeks after adding it because I went back to the project to make some changes. I used a color picker on the design mockups, then searched my tailwind config for `#285e97` (with cmd+f) to figure out the color name I had used, which of course turned up nothing. But yeah otherwise it's not discernible :)

I just swapped the two, but figured I'd report it anyway.
